ci/prebuilt/BUILD

```diff
@@ -101,6 +111,7 @@ cc_library(
         "thirdparty/spdlog/include/**/*.h",
     ]),
     includes = ["thirdparty/spdlog/include"],
+    deps = ["fmtlib"],
```

Comment:
Does this work? I'm guessing it will fail CI, since `googletest` will be used actively, as it's a dependency of most tests, which will then try and resolve `fmtlib`, which will be missing in the existing CI image.

Reply:
I didn't have any problems with this when I ran bazel.debu. I think the line you highlighted is under `spdlog`, not `googletest`?

Reply:
Yeah, `spdlog` it is actually. Still surprised this isn't an issue, but since there is no code dependency, maybe Bazel just finds an empty set for the `.{cc,h}` globs and there is no issue.